android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/navigation/Navigation.kt

```diff
@@ -0,0 +1,23 @@
+package co.nimblehq.ic.kmm.suv.android.ui.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import co.nimblehq.ic.kmm.suv.android.ui.screens.main.MainScreen
+import co.nimblehq.ic.kmm.suv.android.ui.screens.splash.SplashScreen
+
+@Composable
+fun Navigation() {
```

Comment:
OPTIONAL: `NavGraph` sounds more familiar with Android's developers 😄

Reply:
Nice catch!

Fixed in https://github.com/suho/kmm-ic/pull/79/commits/30bef640c4329f8860ab9d30b9e6134791383f58